docs/DeveloperGuide.md

```diff
@@ -307,7 +307,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   * 4a2. WIFE asks if the user wish to add the new tags into the tag list.
     Use case resumes at step 2.
 
+**Use case: Update a food item**
```

Comment:
Can update the use case to follow the numbering of the use cases!

Reply:
rectified!